<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvPACPluginSettingsWidget</class>
 <widget class="QWidget" name="QvPACPluginSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="pacGroupBox">
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="title">
      <string>PAC Settings</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_11">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_74">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>The system proxy will be configured to use the PAC instead of HTTP and SOCKS.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_57">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="pacPortSB">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8088</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_56">
        <property name="text">
         <string>Local IP for PAC</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="pacProxyTxt">
        <property name="placeholderText">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_63">
        <property name="text">
         <string>Use Proxy</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="pacProxyCB">
        <item>
         <property name="text">
          <string>HTTP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SOCKS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_58">
        <property name="text">
         <string>Import GFWList</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="gfwListCB">
          <item>
           <property name="text">
            <string>Mirror: Gitlab</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Github</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirror: Pagure</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirror: Repo.or.cz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirror: Bitbucket</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirror: TuxFamily</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GFWList File</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="getGFWListWithProxyCB">
          <property name="text">
           <string>Download with System Proxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pacGoBtn">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_62">
        <property name="text">
         <string>Edit PAC</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Open PAC Folder</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_66">
        <property name="text">
         <string>PAC Access Path</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="pacListenAddrLabel">
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Listening Address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="pacListenAddrTxt">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
